draft-ietf-httpbis-digest-headers.md

```diff
@@ -219,28 +250,42 @@ The term "checksum" describes the output of the application of an algorithm
 to a sequence of bytes,
 whereas digest is only used in relation to the value of the fields.
 
-# The Digest Field {#digest}
+# Representation Digest {#representation-digest}
```

Comment:
This section is back again :DDD

Reply:
I know we merged it as part of WGLC feedback but I think it helps to restore it when Represenation concept applies to `Digest` and `Representation-Digest` equally.